package.json

```diff
@@ -41,7 +41,7 @@
     "args": "^5.0.1",
     "aws-sdk": "^2.1371.0",
     "axios": "^1.4.0",
-    "bcrypt": "^5.1.0",
+    "bcrypt": "^5.0.1",
```

Comment:
avoid downgrading bcrypt -- this change likely wouldn't affect anything anyway, as `5.1.0` satisfies `^5.0.1`.

Reply:
overrode in package.json